superset/models/dashboard.py

```diff
@@ -407,3 +408,30 @@ def clear_dashboard_cache(
     sqla.event.listen(TableColumn, "after_update", clear_dashboard_cache)
     sqla.event.listen(DruidMetric, "after_update", clear_dashboard_cache)
     sqla.event.listen(DruidColumn, "after_update", clear_dashboard_cache)
+
+
+def get_dashboard(id_or_slug: str) -> Dashboard:
```

Comment:
This is a good candidate for class method. In fact, it would be nice to add a `get_by_id_or_slug` to all entities with slug support for simplicity and consistency. Imagine the following API:

```python
Dashboard.get_by_id(111)
Dashboard.get_by_id_or_slug(xxxx)

# or simply
Dashboard.get(id_or_slug)
Chart.get(id_or_slug)
Tag.get(id_or_slug)
```

Reply:
I agree on the classmethod .
there are no other Entities that has slug